Remotion/Web/Core/ExecutionEngine/WxeLauncher.cs

```diff
@@ -39,7 +39,7 @@ public class WxeLauncher: IHttpHandler
   /// <param name="queryString"> The original query string. </param>
   /// <returns> The query string to pass on the <see cref="WxeHandler"/>. </returns>
   /// <remarks> Override this method to change the query string. </remarks>
-  protected virtual string GetQueryString (NameValueCollection queryString)
+  protected virtual string? GetQueryString (NameValueCollection queryString)
```

Comment:
I verified that this cannot return `null`. Still I wonder what the purpose of this implementation is, as it would just return `typeof(NameValueCollection).ToString()` when not overridden.

Reply:
Oh I just read your other comment where you're wondering the same.